client/src/components/Header/Header.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./Header.css";
+import InputField from "../InputField/InputField";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
This is too much dependency on font-awesome. This can be simply achieved by adding a `<i></i>` tag for icons. Also, we are now loading entire font-awesome NodeJS packages, which depends on fonts, lots of svg, which will be loaded from our app. Instead, using CDN based font-awesome CSS link would make more sense.

Reply:
will update that

Reply:
Removed this